app-nodejs-html5/src/main/webapp/index.html

```diff
@@ -0,0 +1,50 @@
+<!--
+    JBoss, Home of Professional Open Source
+    Copyright 2016, Red Hat, Inc. and/or its affiliates, and individual
+    contributors by the @authors tag. See the copyright.txt in the
+    distribution for a full listing of individual contributors.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Keycloak Example App</title>
+
+    <link rel="stylesheet" type="text/css" href="styles.css"/>
+
+    <script src="keycloak.js"></script>
+    <script src="app.js"></script>
+</head>
+
+<body>
+    <div class="wrapper">
+        <div id="not-authenticated" class="menu">
+            <button name="loginBtn" onclick="keycloak.login()">Login</button>
```

Comment:
Event listeners should be added by the JavaScript included by the page (as this code now relies on globals). For example in `app.js` you could add:

```js
function onLogin() {
  keycloak.login();
}

const loginButton = document.getElementById('loginBtn');

loginBtn.addEventListener('click', onLogin);
```

Reply:
@jonkoops all of the webapp code was simply copied from https://github.com/keycloak/keycloak-quickstarts/tree/latest/app-jee-html5/src/main/webapp, they should likely be consistent so maybe all of those changes should be made to both copies in 1 PR?